docs/en/pact-reference.md

```diff
@@ -1277,50 +1277,67 @@ Declaring models shares the same syntax with modules:
 Module References {#modrefs}
 ---
 
-Pact 3.7 introduces module references (also called "modrefs"), a new language feature that enables important use-cases
-that require polymorphism. For example, a Uniswap-like DEX allows users to specify pairs of tokens to allow trading
-between them. The `fungible-v2` interface allows tokens to offer identical operations such as `transfer-create`, but without
-a way to abstract over different `fungible-v2` implementations, a DEX smart contract would have to be upgraded for each
-pair with custom code for every operation.
-
-With module references, the DEX can now accept pairs of modref values where each value references a concrete module that
-implements the `fungible-v2` interface, giving it the ability to call `fungible-v2` operations using those values. The refmod
-values are "normal Pact values" that can be stored in the database, referenced in events and returned from
-functions.
+Pact 3.7 introduces module references (also called "modrefs"), a new language
+feature that enables important use-cases that require polymorphism. For example,
```

Comment:
Since the primary reader is not a Haskeller, I doubt this terminology would be clarifying in any way.

Reply:
I agree; I suggest we even go a step further and quickly specify how module references work (even though the example goes into more detail). For example, a quick sentence:

```suggestion
feature that enables important use-cases that require polymorphism. Specifically, module references allow Pact code to use functions, values, and capabilities as defined in a particular interface and defer choosing a concrete implementation from a particular module. Instead of using the `coin.transfer-create` function directly, for example, Pact code can use the `transfer-create` function from _any_ module that implements the `fungible-v2` interface. Consider
```